types/src/collection.rs

```diff
@@ -0,0 +1,15 @@
+use uuid::Uuid;
+use {
+    crate::*,
+};
+
+#[derive(Debug, Clone)]
+pub struct Collection {
+    pub title: String,
+
+    pub wordpress_menu_id: i64,
```

Comment:
Maybe make this a `Option<i64>` since like `wordpress_id` in chapter the sql allows null as a possibility.